client/src/components/PropertyDetails.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const PropertyDetails = props => {
+  return (
+    <div className="container">
+      <ul className="collection ">
+        <li className="collection-item">
+          <img
+            className="materialboxed center"
+            height="200"
+            width="300"
+            src={props.details.imageUrl}
+            alt=""
+          />
+        </li>
+        <li className="collection-item">
+          <div className="">Price in NIS:{props.details.price}</div>
+        </li>
+        <li className="collection-item">
+          <div className="">
+            USD:{props.details.price && Math.round(props.details.price / 3.42)}
```

Comment:
The exchange rate is not a constant...

Reply:
I have changed. Now I am using an API to get the current and latest exchange rates in real time.